ramls/loan.json

```
@@ -21,6 +21,10 @@
       "description": "ID of the item lent to the patron",
       "type": "string"
     },
+    "itemLocationId": {
```

Under what circumstances is this the item location ID?

For example, is this the effective location at the point of check in, check out or the current one?

---

Based upon the description, this is the effective location at the point of check out. Please could it be renamed to reflect that.

```suggestion
    "itemEffectiveLocationAtCheckOut": {
```

---

Is this the only property we are intending to store for reporting purposes? 

If other parts of the item location is intended to be kept, or other properties about the item, it might be worth created a more nested structure.

---

Should we have the UUID pattern applied to this field? It is part of other IDs in this file and the description states that it is a UUID.